### PR TITLE
Output the accepted task types of each window in sched-diag

### PR DESCRIPTION
**Pull request by lanzafame**
@magik6k I created this against the wrong branch, but I am looking for feedback on how to get this to work. I want to have the accepted task types for each window next to them, so that it is possible to know which are missing

### CODE REVIEW

File: extern/sector-storage/sched.go [hunk 1]
```diff
@@ -204,9 +204,14 @@ type SchedDiagRequestInfo struct {
 	Priority int
 }
 
+type SchedDiagWorkerInfo struct {
+	WorkerID
+	AcceptedTaskTypes map[sealtasks.TaskType]struct{}
```
Comment (magik6k) [inline extern/sector-storage/sched.go:209]: Could just be `[]sealtasks.TaskType`

File: extern/sector-storage/sched.go [hunk 2]
```diff
@@ -256,10 +261,21 @@ func (sh *scheduler) diag() SchedDiagInfo {
 		})
 	}
 
+	var workerInfo []SchedDiagWorkerInfo
+	sh.workersLk.RLock()
```
Comment (magik6k) [inline extern/sector-storage/sched.go:265]: missing unlock?

Comment (lanzafame) [reply extern/sector-storage/sched.go:265]: Yes! thanks

File: extern/sector-storage/sched.go (hunk 1 above)
Comment (magik6k) [reply extern/sector-storage/sched.go:209]: (bonus points for having it sorted)

Comment (lanzafame) [reply extern/sector-storage/sched.go:209]: I did that originally, but this means I don't have to transform the output of `tt, err := sh.workers[window.worker].w.TaskTypes(context.TODO())` Happy to change it back

**Comment by lanzafame** [comment]
@magik6k This has been squashed and rebased onto `master`.

**Comment by magik6k** [comment]
(stale)

**Pull request closed without merging**